.gitignore

```diff
@@ -28,3 +28,6 @@ yarn-error.log*
 .env.development.local
 .env.test.local
 .env.production.local
+
+# ides
+.vscode
```

Comment:
i have projects where I use 4 spaces per tab.  Had to tell VSCode this project only uses 2.  So, I ignore the .vscode folder.